README.md

```diff
@@ -1 +1,26 @@
-# halcyonsim
\ No newline at end of file
+# HalcyonSim
+
+Simulates a virtual world environment. With sun, moon, clouds, land, buildings *(objects with seamless textures to give the desired look)*, avatars.. and interactive objects that are scripted.
```

Comment:
Fragmented sentence. Please edit.
Was that `..` supposed to be a comma?

Reply:
Please re-read the updated version and let me know if there's still fragmented sentence(s) still and have removed the double dots in the new version. Thank you.